src/keeshare/CMakeLists.txt

```diff
@@ -15,9 +17,17 @@ if(WITH_XC_KEESHARE)
     )
 
     add_library(keeshare STATIC ${keeshare_SOURCES})
-    if(WITH_XC_KEESHARE_SECURE)
-        target_link_libraries(keeshare Qt5::Core Qt5::Widgets ${GCRYPT_LIBRARIES} ${QUAZIP_LIBRARIES} ${crypto_ssh_LIB})
-    else()
-        target_link_libraries(keeshare Qt5::Core Qt5::Widgets ${GCRYPT_LIBRARIES} ${crypto_ssh_LIB})
+    target_link_libraries(keeshare PUBLIC Qt5::Core Qt5::Widgets ${GCRYPT_LIBRARIES} ${crypto_ssh_LIB})
+
+    # Try to find libquazip5, if found, enable secure sharing
+    find_package(QuaZip QUIET)
+    if(QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
+        message(STATUS "Found QUAZIP: building with KeeShare secure container support")
```

Comment:
I would remove the QUIET option and only print a warning message if QUAZIP_FOUND is not set (and you should definitely check that variable instead of INCLUDE_DIRS or LIBRARIES in any case). We don't need to print anything if Quazip was indeed found.